src/eth/erc4337/types.test.ts

```diff
@@ -76,4 +76,73 @@ describe('types', () => {
       'At path: nonce -- Expected a value of type `EthUint256`, but received: `"0x01"`',
     );
   });
+
+  describe('EthBaseUserOperationStruct', () => {
+    it('is a valid BaseUserOperation', () => {
+      const userOp = {
```

Comment:
What about adding a:

```typescript
const baseUserOp = {
  nonce: '0x1',
  initCode: '0x',
  ...
};
```

and spread it to `userOp` like:

```typescript
const userOp = {
  ...baseUserOp,
  bundlerUrl: 'https://examplke.com',
}
```

That would be a bit more dry and make the actual "test input" (the `bunderUrl` here) easier to spot/read.

We could use in all 3 test cases here.

WDYT?

Reply:
Sure, changed in f460710374c8b1d6d8118ce6ff71cfe2cad3a2fa